src/main/java/TaskManager.java

```diff
@@ -0,0 +1,235 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskManager {
+    /**
+     * list to store all tasks
+     */
+    private final static List<Task> tasks = new ArrayList<>();
+
+    /**
+     * Add all the tasks in args
+     *
+     * @param args tasks to be added
+     */
+    public static void addTasks(String[] args) {
+        if (!args[0].equals("add")) {
+            // there must be some error
+            // TODO exception handle
+            return;
+        }
+        for (int i = 1; i < args.length; i++) {
+            Task task = new Task(args[i]);
+            tasks.add(task);
+            System.out.println("Added: " + task);
+        }
+    }
+
+    /**
+     * Return the corresponding task index by its array index.
+     * By conventional task index == array index + 1
+     *
+     * @param arrayID array id
+     * @return task id of corresponding array id
+     */
+    private static int getTaskID(int arrayID) {
+        return arrayID + 1;
+    }
+
+    /**
+     * Return the corresponding task index by its array index.
+     * By conventional array index == task index - 1
+     *
+     * @param taskID task id
+     * @return array id of corresponding task id
+     */
+    private static int getArrayID(int taskID) {
+        return taskID - 1;
+    }
+
+    /**
+     * List all the tasks
+     *
+     * @param args dump variable, no use
+     */
+    public static void listTasks(String[] args) {
+        // TODO args can be used to control the style of output
+        if (!args[0].equals("list")) {
+            // there must be some error
+            // TODO exception handle
+            return;
+        }
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.printf("%d. %s\n", getTaskID(i), tasks.get(i));
+        }
+    }
+
+    /**
+     * Mark every task whose is in args[1..n) as done
+     * Using by > mark {task1 id} {task2 id} ...
+     *
+     * @param args arguments
+     */
+    public static void mark(String[] args) {
+        if (!args[0].equals("mark")) {
+            // there must be some error
+            // TODO exception handle
+            return;
+        }
+        System.out.println("Nice! I've marked these tasks as done\n");
+        for (int i = 1; i < args.length; i++) {
+            int taskID = Integer.parseInt(args[i]);
+            int arrayID = getArrayID(taskID);
+            if(arrayID < 0 || arrayID >= tasks.size()) {
```

Comment:
can be better if its in a seperate function

Reply:
> can be better if its in a seperate function

Yup, I'll try to extract a method.